src/persist/src/error.rs

```diff
@@ -43,6 +45,7 @@ impl fmt::Display for Error {
             Error::IO(e) => fmt::Display::fmt(e, f),
             Error::OutOfQuota(e) => f.write_str(e),
             Error::String(e) => f.write_str(e),
+            Error::UnknownRegistration(id) => write!(f, "Unknown registration: {}", id),
```

Comment:
nit: lower-case U in "unknown" (most of persist uses lowercase for this, looks like it was "wrong" before your change)

Reply:
👌